src/Domains.py

```diff
@@ -0,0 +1,25 @@
+from dotenv import load_dotenv
+import os
+
+load_dotenv()
+
+network = "xdai"
```

Comment:
I see that we `load_dotenv()` and use it to fetch the `PUBLIC_ADDRESS` and `PRIVATE_KEY` so we could also load the network here with `network = os.getenv("NETWORK")`

Reply:
added it!